packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -26,7 +26,7 @@ export function isTreeItemElement(el: Element): el is HTMLElement {
  * @public
  */
 export type TreeItemOptions = StartEndOptions & {
-    expandCollapseGlyph?: string | SyntheticViewTemplate;
+    expandCollapseIcon?: string | SyntheticViewTemplate;
```

Comment:
Make non optional after above change.

Reply:
The icon? Per the other discussion I think we're leaving them all optional now.